README.md

```diff
@@ -93,6 +103,7 @@ This plugin use the new [miio](https://github.com/aholstenson/miio) version 0.15
 | `waterBox` | false | when set to true, HomeKit shows an additional slider to control the amount of water released by the robot (only selected models like S5-Max). Currently in a beta state. |
 | `cleanword` | cleaning | used for autonaming the Roomselectors |
 | `rooms` | false | Array of ID / Name for a single Room. If set you have another switch for cleaning only this room |
+| `zones` | false | Array of name / zone coordinates for a single zone. Coordinates are laid out as bottom-left-x, bottom-left-y, top-right-x, top-right-y, number-of-cleanings.  A separate tile in Home will be created for each zone.  Figuring out coordinates will take some trial and error.  Ensure this section is surrounded with double brackets: [[ ]]
```

Comment:
> Ensure this section is surrounded with double brackets: [[ ]]

This is because we are now supporting multiple _squares_ to be considered one zone group. Alas: we can say:
```
{
  "zones": [
    {
      "name":"Family Room (x2)",
      "zone":[[25000,25000,32000,32000,2]]
    },
    {
      "name":"Bedroom",
      "zone":[[21000,32000,24000,37000,1]]
    },
    {
      "name":"Bedroom & Family Room",
      "zone":[ [21000,32000,24000,37000,1],  [25000,25000,32000,32000,1]]
    }
  ]
}
```

Can we update the readme to make sure it's clear?

Reply:
Good catch on the zone groups.  I havent actually tested out this out.  The readme was updated.